escuela-nacional-de-antropologia-e-historia-full-note

```diff
@@ -0,0 +1,595 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="es-MX">
+  <info>
+    <title>Escuela Nacional de Antropología e Historia (nota completa)</title>
+    <title-short>ENAH (nota completa)</title-short>
+    <id>http://www.zotero.org/styles/escuela-nacional-de-antropologia-e-historia-full-note</id>
+    <link href="http://www.zotero.org/styles/escuela-nacional-de-antropologia-e-historia-full-note" rel="self"/>
+    <link href="http://www.enah.edu.mx/index.php/difu-cul/publicaciones/normas-editoriales.pdf" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="template"/>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+    <author>
+      <name>Juan Ignacio Flores Salgado</name>
+      <email>natch.nacht@gmail.com</email>
+      <uri>https://www.mendeley.com/profiles/juan-ignacio-flores-salgado/</uri>
+    </author>
+    <contributor>
+      <name>Inés Segovia Camelo</name>
+      <email>publicaciones.enah@inah.gob.mx</email>
+      <uri>http://www.enah.edu.mx/index.php/difu-cul/publicaciones</uri>
+    </contributor>
+    <category citation-format="note"/>
+    <category field="anthropology"/>
+    <category field="history"/>
+    <summary>Estilo de citas de la Escuela Nacional de Antropología e Historia -- variante nota completa</summary>
+    <updated>2015-06-25T20:21:07-06:00</updated>
+    <!--ISSN revista Cuicuilco-->
+    <issn>1405-7778</issn>
```

Comment:
This CSL file is going to be used in all the publication types at school, including the journal.


--
Enviado desde mi iMac

> El 28/06/2015, a las 1:16 a. m., Sebastian Karcher <notifications@github.com> escribió:
> 
> In escuela-nacional-de-antropologia-e-historia-full-note <https://github.com/citation-style-language/styles/pull/1603#discussion_r33419293>:
> 
> > +      <name>Juan Ignacio Flores Salgado</name>
> > +      <email>natch.nacht@gmail.com</email>
> > +      <uri>https://www.mendeley.com/profiles/juan-ignacio-flores-salgado/</uri>
> > +    </author>
> > +    <contributor>
> > +      <name>Inés Segovia Camelo</name>
> > +      <email>publicaciones.enah@inah.gob.mx</email>
> > +      <uri>http://www.enah.edu.mx/index.php/difu-cul/publicaciones</uri>
> > +    </contributor>
> > +    <category citation-format="note"/>
> > +    <category field="anthropology"/>
> > +    <category field="history"/>
> > +    <summary>Estilo de citas de la Escuela Nacional de Antropología e Historia -- variante nota completa</summary>
> > +    <updated>2015-06-25T20:21:07-06:00</updated>
> > +    <!--ISSN revista Cuicuilco-->
> > +    <issn>1405-7778</issn>
> what's up with the ISSN? If this isn't for that journal it shouldn't be included here.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/citation-style-language/styles/pull/1603/files#r33419293>.
>